src/definition/scheduler/IRecurringSchedule.ts

```diff
@@ -1,5 +1,12 @@
+/** Represents a job that runs recurrently */
 export interface IRecurringSchedule {
+    /** The schedule's identifier */
     id: string;
+    /**
+     * When the job will be run. Must be a
+     * [human-interval](https://github.com/agenda/human-interval)
+     */
```

Comment:
The name of this property is cron yet you're referencing a `human-interval` and I can not find any referencing in that package's readme to cron.

Reply:
I think this is a case of updating the documentation of this prop. We can attribute cron strings to it, as well as strings handled by `human-interval`, but I failed to add this in the comment